pkg/util/configblob/container.go

```diff
@@ -11,10 +14,34 @@ import (
 	"github.com/openshift/openshift-azure/pkg/util/cloudprovider"
 )
 
+type StorageKey struct {
+	Name string `json:"name"`
+	Key  string `json:"key"`
+}
+
 // GetService is a helper function called by code running outside of the plugin.
 // It returns the blob storage interface to the storage account containing the
 // config blob, etcd backups, etc.
-func GetService(ctx context.Context, cpc *cloudprovider.Config) (azureclientstorage.BlobStorageClient, error) {
+func GetService(cloudproviderDir string) (azureclientstorage.BlobStorageClient, error) {
```

Comment:
can you drop this change for now and stick with using ListKeys?

Reply:
sure can do